src/docfx/build/xref/XrefMap.cs

```diff
@@ -83,12 +107,14 @@ private static void Load(Context context, ConcurrentDictionary<string, Concurren
                 var content = file.ReadText();
                 if (file.FilePath.EndsWith(".md", PathUtility.PathComparison))
                 {
-                    var (metaErrors, metadata) = ExtractYamlHeader.Extract(file, context);
-                    errors.AddRange(metaErrors);
-                    var uid = metadata.Value<string>("uid");
-                    if (!string.IsNullOrEmpty(uid))
+                    var (yamlHeaderErrors, yamlHeader) = ExtractYamlHeader.Extract(file, context);
+                    var (metaErrors, metadata) = JsonUtility.ToObject<FileMetadata>(yamlHeader);
```

Comment:
>metadata [](start = 37, length = 8)

yamlheader is not enough?
should merge with `filemetadata` and `globalMetadata`?

Reply:
Here we only care about the metadata within the yaml header, right?